client/lib/accept/test/index.js

```diff
@@ -48,15 +47,8 @@ describe( '#accept()', () => {
 	} );
 
 	test( 'should clean up after itself once the prompt is closed', () => {
-		accept( 'Are you sure?', function() {
-			jest.useFakeTimers();
-			process.nextTick( function() {
-				expect( document.querySelector( '.accept-dialog' ) ).to.be.null;
-			} );
-			jest.runAllTimers();
-			jest.useRealTimers();
-		} );
-
+		accept( 'Are you sure?', () => {} );
 		document.querySelector( '.button.is-primary' ).click();
+		expect( document.querySelector( '.accept-dialog' ) ).to.be.null;
```

Comment:
It was surprisingly difficult to come to this simple solution this test fix.

Reply:
Lol, should work as expected when you compare with one of the previous tests :)